packages/loader/src/gltf/parser/GLTFParser.ts

```
@@ -23,15 +23,15 @@
     context: GLTFParserContext,
     ownerSchema: GLTFExtensionOwnerSchema,
     ...extra
-  ): EngineObject | void | Promise<EngineObject | Uint8Array | void> {
-    let resource: EngineObject | Promise<EngineObject> = null;
+  ): EngineObject | void | AssetPromise<EngineObject | Uint8Array | void> {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)